gendoc/src/main.rs

```diff
@@ -0,0 +1,123 @@
+//! Generate documentation for a ciphersuite based on another ciphersuite implementation.
+//!
+//! The documentation for each ciphersuite is very similar, with the only difference being
+//! the ciphersuite name.
+//!
+//! To make it easier to update all ciphersuite documentation when a change is needed,
+//! this binary allows updating all of them based on a single one. This binary
+//! uses frost-ristretto255 as the "canonical" one, so:
+//!
+//! - Change any documentation of a public function or struct in `frost-ristretto255/src/lib.rs`
+//! - Run `cargo run --manifest-path gendoc/Cargo.toml` to update the documentation
+//!   of the other ciphersuites.
+
+use std::{fs, iter::zip};
+
+use regex::Regex;
+
+/// Read the public documentation of public symbols in the given file.
```

Comment:
```suggestion
/// Read the public documentation of public types in the given file.
```

Reply:
I think this is correct?

Reply:
I was looking for a word that covers types, functions, constants...

Reply:
Perhaps 'items': https://doc.rust-lang.org/reference/items.html

Reply:
Ah neat, thanks!

Reply:
Replaced in aa95773